sos/plugins/lustre.py

```diff
@@ -51,4 +51,10 @@ def setup(self):
         self.get_params("quota", ["osd-*.*.quota_slave." +
                                   "{info,limit_*,acct_*}"])
 
+        # Grab emergency ring buffer dumps
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/tmp/lustre-log.*"
```

Comment:
What causes this file to be written? Is it present at all times, or is some action required to cause it to be present?